.drone.yml

```diff
@@ -7,6 +7,8 @@ pipeline:
     image: ncatelli/golang:1.10.2-alpine3.7
     commands:
       - make
```

Comment:
So the make depends on all dependencies being installed. So those commands will need to be run first.

```
+ make
go fmt ./...
go test ./...
command/version/version.go:6:2: cannot find package "github.com/mitchellh/cli" in any of:
	/usr/local/go/src/github.com/mitchellh/cli (from $GOROOT)
	/go/src/github.com/mitchellh/cli (from $GOPATH)
lib/config/config.go:6:2: cannot find package "gopkg.in/yaml.v2" in any of:
	/usr/local/go/src/gopkg.in/yaml.v2 (from $GOROOT)
	/go/src/gopkg.in/yaml.v2 (from $GOPATH)
lib/drivers/mysql/mysql.go:6:2: cannot find package "github.com/go-sql-driver/mysql" in any of:
	/usr/local/go/src/github.com/go-sql-driver/mysql (from $GOROOT)
	/go/src/github.com/go-sql-driver/mysql (from $GOPATH)
make: *** [Makefile:10: test] Error 1
```

One way I'd recommend approaching this would be to not add the `glide update` and `glide install` directly into the drone file but instead add it to the makefile.